modules/imgproc/src/colormap.cpp

```diff
@@ -758,6 +780,7 @@ namespace colormap
             colormap == COLORMAP_TWILIGHT ? (colormap::ColorMap*)(new colormap::Twilight) :
             colormap == COLORMAP_TWILIGHT_SHIFTED ? (colormap::ColorMap*)(new colormap::TwilightShifted) :
             colormap == COLORMAP_VIRIDIS ? (colormap::ColorMap*)(new colormap::Viridis) :
+            colormap == COLORMAP_DEEPGREEN ? (colormap::ColorMap*)(new colormap::DeepGreen) :
```

Comment:
Minor.
Can you keep it sorted alphabetically?

Reply:
you are right,I have changed my code.

Reply:
#15388 is great!many thanks!@catree